cmd/registry/cmd/apply/apply.go

```diff
@@ -25,16 +25,27 @@ import (
 )
 
 func Command() *cobra.Command {
-	var fileName string
+	var files []string
 	var project string
 	var recursive bool
 	var jobs int
 	cmd := &cobra.Command{
 		Use:   "apply (-f FILE | -f -)",
 		Short: "Apply YAML to the API Registry",
-		Long:  "Apply YAML to the API Registry by file name or stdin. Resources will be created if they don't exist yet.\n\nMore info and example usage at https://github.com/apigee/registry/wiki/registry-apply",
-		Args:  cobra.NoArgs,
+		Long: "Apply YAML to the API Registry by files / folder names or stdin. " +
+			"Resources will be created if they don't exist yet. " +
+			"Multiple files may be specified in -f separated by commas or by repeating the -f flag." +
```

Comment:
Does `kubectl apply -f` split on commas? I didn't know about that.

Reply:
No idea if kubectl does, but this does.

Reply:
If kubectl doesn't do it, would you mind removing that? It's duplicative of multiple `-f` options, so it's only a small usability improvement.

Reply:
I'll remove the comment, but that's just the way that cobra works.